src/app/fyle/add-edit-expense/add-edit-expense.page.ts

```
@@ -1308,6 +1315,60 @@
     }
   }
 
+  updateEtxnWithExtractedData(
+    etxn: Partial<UnflattenedTransaction>,
+    imageData: InstaFyleResponse,
+    homeCurrency: string,
+    extractedCategory: OrgCategory
+  ): void {
+    const extractedData = {
+      amount: imageData && imageData.parsedResponse && imageData.parsedResponse.amount,
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)